docs/pages/index.md

```diff
@@ -0,0 +1,51 @@
+# SKNNR Docs
+
+`scikit-learn-knn-regression` (a.k.a. `sknnr`) is a package for running k-nearest neighbor imputation methods, including GNN (Ohmann & Gregory, 2002), MSN (Moeur & Stage 1995), and RFNN (Crookston & Finley, 2008), using estimators that are fully compatible with [`scikit-learn`](https://scikit-learn.org/stable/).
+
+## Quick-Start
+
+1. Follow the [installation guide](installation.md).
+2. Import any `sknnr` estimator, like [MSNRegressor](api/estimators/msn.md), as a drop-in replacement for a `scikit-learn` regressor.
+```python
+from sknnr import MSNRegressor
+
+est = MSNRegressor()
+```
+3. Load a custom dataset like [SWO Ecoplot](api/datasets/swo_ecoplot.md) (or bring your own).
+```python
+from sknnr.datasets import load_swo_ecoplot
+
+X, y = load_swo_ecoplot(return_X_y=True, as_frame=True)
+```
+4. Train, predict, and score [as usual](https://scikit-learn.org/stable/getting_started.html#fitting-and-predicting-estimator-basics).
+```python
+from sklearn.model_selection import train_test_splot
+
+X_train, X_test, y_train, y_test = train_test_split(X, y)
+
+est = est.fit(X_train, y_train)
+est.score(X_test, y_test)
+```
+5. Check out the additional features like [independent scoring](usage.md/#independent-scores-and-predictions), [dataframe indexing](usage.md/#retrieving-dataframe-indexes), and [dimensionality reduction](usage.md/#dimensionality-reduction).
+```python
+# Evaluate the model using the second-nearest neighbor in the test set
+print(est.fit(X, y).independent_score_)
+
+# Get the dataframe index of the nearest neighbor to each plot
+print(est.kneighbors(return_dataframe_index=True, return_distance=False))
+
+# Apply dimensionality reduction using CCorA ordination
+MSNRegressor(n_components=3).fit(X_train, y_train)
+```
+
+## Acknowledgements
+
+`sknnr` was inspired by the [yaImpute](https://cran.r-project.org/web/packages/yaImpute/index.html) package for R (Crookston & Finley 2008). Thanks to Andrew Hudak for the inclusion of the [Moscow Mountain / St. Joes dataset](api/datasets/moscow_stjoes.md) (Hudak 2010), and Tom DeMeo for the inclusion of the [SWO Ecoplot dataset](api/datasets/swo_ecoplot.md) (Atzet et al., 1996). Development of this package was funded in part by an appointment to the United States Forest Service (USFS) Research Participation Program administered by the Oak Ridge Institute for Science and Education (ORISE) through an interagency agreement between the U.S. Department of Energy (DOE) and the U.S. Department of Agriculture (USDA).
```

Comment:
As with the readme, please edit!

Reply:
This is really a great first cut, I think.  I'll suggest some edits here shortly.

Reply:
Please feel free to edit!  As I mentioned, `yaImpute` is now in the "History and Inspiration" section and I've added my grants in acknowledgements.

Reply:
Looks perfect!